app/Http/Controllers/LoginController.php

```diff
@@ -128,10 +134,15 @@ public function authenticateDiscord(Request $request): RedirectResponse {
         if (sizeof($staffMemberSelected) > 0) {
             // It's a valid staff member, we need to log them in
             Auth::guard('web')->loginUsingId($staffMemberSelected['staff_id']);
-            return redirect()->intended('DASHBOARD');
+            $installerController = './InstallerController.php';
```

Comment:
Instead of duplicated code here, you should have a function that does this and then just use in both places 😄